bundle/manifests/externaldns.olm.openshift.io_crd.yaml

```diff
@@ -77,7 +77,7 @@ spec:
                       - Exact
                       - Pattern
                       type: string
-                    names:
+                    name:
```

Comment:
Hm, this one looks familiar :)
https://github.com/openshift/external-dns-operator/pull/55/files

Reply:
yeah but then git shall not show this difference